frontend/javascripts/oxalis/view/layouting/tracing_layout_view.tsx

```diff
@@ -169,6 +178,37 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
     });
     initializeInputCatcherSizes();
     window.addEventListener("resize", this.debouncedOnLayoutChange);
+    window.addEventListener("touchstart", this.handleTouch);
+    window.addEventListener("mouseover", this.handleMouseOver, false);
+
+    if (window.screen.width <= 1080) {
+      // Simply assume mobile.
+      const { left, right } = Store.getState().uiInformation.borderOpenStatus;
+      if (left) {
+        layoutEmitter.emit("toggleBorder", "left");
+      }
+      if (right) {
+        layoutEmitter.emit("toggleBorder", "right");
+      }
+      // Immediately show mobile buttons
+      this.handleTouch();
+    }
+  };
+
+  handleTouch = () => {
+    this.lastTouchTimeStamp = Date.now();
+    return this.setState({ showFloatingMobileButtons: true });
+  };
+
+  handleMouseOver = () => {
+    if (this.lastTouchTimeStamp && Date.now() - this.lastTouchTimeStamp < 1000) {
+      // Ignore mouse move events when they are shortly after touch events because the browser
+      // emulates these events when touch is used.
+      // Also ignore the event when touch was never used, because then the mobile buttons
+      // were never shown, anyway.
+      return;
+    }
+    return this.setState({ showFloatingMobileButtons: false });
```

Comment:
Minor, but I would set `this.lastTouchTimeStamp = null` here as well. Otherwise, `setState` will be called on every subsequent mouseOver event if there ever was a touch event.